code/style.css

```diff
@@ -1,49 +1,68 @@
-/* Global css variables used for colors */
-:root {
-  --primary: #a259ff;
-  --secondary: #b0a6ff;
-}
 
 body {
   display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
   margin: 0;
   padding: 0;
-  font-family: 'Montserrat', sans-serif;
+  background-color: #b39660;
+  font-family: 'Special Elite', cursive;
+}
+
+.header-container {
+  width: 220px;
+  display: flex;
+  justify-content:center;
 }
 
 h1 {
-  font-size: 42px;
+  font-size: 35px;
+}
+
+h2 {
+  font-size: 30px;
   font-weight: 500;
-  line-height: 48px;
-  margin: 10px 0;
-  color: white;
+  line-height: 30px;
+  margin: 20px 10px 0 10px;
+  color: black;
 }
 
 .question-section {
+  min-width:300px;
   width: 30%;
-  min-height: 100vh;
-  background-color: var(--secondary);
+  height: 100vh;
   display: flex;
   flex-direction: column;
   padding: 5vw;
-  align-items: flex-start;
+  align-items: center;
   box-sizing: border-box;
 }
 
+.notesheet {
+  background-image: url("./images2/notesheet.svg");
+  display: block;
+  box-shadow: 3px 3px 5px #4d4028;
+  width: 300px;
+  height: 500px;
+  margin: 20px;
+}
```

Comment:
Really liked this, nice touch

Reply:
Glad to here it! I made the svg myself here https://editor.method.ac/